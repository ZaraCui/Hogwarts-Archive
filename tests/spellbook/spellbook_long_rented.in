ADD COLLECTION tests/data/books_small.csv
ADD STUDENT Harry Potter
RENT 100000 1003
SPELLBOOK 1003 LONG
EXIT

