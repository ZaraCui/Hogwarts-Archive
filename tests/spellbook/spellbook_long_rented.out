user: 12 spellbooks successfully added.

user: Success.

user: Success.

user: 1003: Patronus Guide (Godric Gryffindor, Defense)
Rented by: 100000.

user: Ending Archive process.
